ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -159,9 +159,7 @@ def verify_drpolicy_ui(acm_obj, scheduling_interval):
     """
     ocp_version = get_ocp_version()
     acm_loc = locators[ocp_version]["acm_page"]
-    acm_obj.navigate_data_services()
-    log.info("Click on 'Policies' tab under Disaster recovery")
-    acm_obj.do_click(acm_loc["Policies"], avoid_stale=True, enable_screenshot=True)
+    acm_obj.navigate_disaster_recovery()
```

Comment:
```suggestion
    acm_obj.navigate_data_services()
```
You can retain the original function which is now compatible with ODF 4.16/ACM 2.11